app/views/search/index.slim

```diff
@@ -16,21 +17,24 @@ article#search-results
   - @results.each do |search_type, results|
     - if results.present?
       h4 = search_type.to_s.capitalize
-      .card.mb-3
-        table.card-body.table.table-striped.mb-0
-          tbody
-            - results.each do |obj|
-              tr
-                td
-                  - case search_type
-                  - when :games
-                    = link_to obj, game_path(obj), class: 'game-link'
-                    small.float-right = obj.categories.pluck(:name).join(', ')
-                  - when :users
-                    = link_to obj, user_path(obj), class: 'game-link'
-                    small.float-right = obj.games.pluck(:shortname).compact.uniq.join(', ')
-
+      .card.mb-3.shadow
+        .table-responsive
```

Comment:
Is there a reason to not have this in a `.card-body` and to leave it on the top level card?

Reply:
Hmm, it seems like in the [`.card` docs][1] they generally use `.card-body` only when they're putting naked text right into the card. For the `.card > .list-group`, `.card > .card-text`, and `.card > .card-img` use cases they put those elements right inside the `.card` with no `.card-body`.

There's no example for tables, but it seems to match that convention of not being naked text, so I'd argue it doesn't belong inside a `.card-body`.

Practically, using a `.card-body` gives this:
<img width="1358" alt="screenshot 2018-06-09 16 55 12" src="https://user-images.githubusercontent.com/438911/41197080-75308244-6c06-11e8-87b7-a767e90ceb4e.png">
whereas not using one gives:
<img width="1256" alt="screenshot 2018-06-09 16 55 15" src="https://user-images.githubusercontent.com/438911/41197090-aee66922-6c06-11e8-9c3a-e06c54a03c6e.png">

and I like the second one more. You could also accomplish this with a `.card-body.p-0`, but at that point I'm not sure what the difference is between that and not using one, so I'd defer back to my original point above.

[1]: https://getbootstrap.com/docs/4.1/components/card/

Reply:
Ah, when I went through the docs it made it seem like you should put stuff in the body portion, but I was probably misinterpretting.  I agree it looks better without it.  Thanks for the info!